jenkins/slave-base/README.md

```diff
@@ -11,3 +11,4 @@ The base jenkins slave used by all ODS [builder slaves](https://github.com/opend
 1. [OWASP scan plugin](https://dl.bintray.com/jeremy-long/owasp/)
 1. Creates proxy awareness when `HTTP_PROXY` is injected during build pahse
 1. [Tailor](https://github.com/opendevstack/tailor) - on top of `openshift CLI` to provide infrastrucure as code
+1. Installs [Snyk Security Scan CLI](https://github.com/snyk/snyk) when 'SNYK_DISTRIBUTION_URL' is injected during build phase
```

Comment:
Use backticks to make it render in code blocks. See difference: 'SNYK_DISTRIBUTION_URL' vs `SNYK_DISTRIBUTION_URL`.

Reply:
@michaelsauter done